text/0000-temp-config-settings.md

```diff
@@ -13,65 +13,368 @@ nit-picking the "gotchas" in code examples. :heart:
+because there is currently no standadard.
+
+For the sake of the argument, let's say that the implementation cost is high.
+
+> :warning: **Note:** This is ultimately the goal to resolve via Pando. The more
+> we can standardize the way we do things, the more we can reduce the implementation
+> cost and the more we can focus on the product.
+
+### User space implementation
+
+Right now, every team uses React across the company (outside of one or two Skills
+BC's) so this is not a concern. Likewise, most teams are using CSS-in-JS so this
+enhancement should be easy to pick up and migrate. For the most part, teams should
+be reducing code by adopting and using Pando React.
+
+Panda :panda: is becoming _very_ popular across the across the industry for this exact
+reason. It is easy to pick up and use.
+
+### Impact on teaching
+
+Documentation shouldn't be too affected by this change. The only thing that would
+change is the way we teach the installation process and how to use Pando React
+via the new customizations it will offer.
+
+### Integration with other features
+
+No different here than just getting teams to use the Pando React library over
+the current chaos. Once again, this is a good thing because it will allow us to
+create a more consistent experience (both customer and developer) across the company.
+
+### Cost of migration
+
+This would introduce a breaking change **only** for teams that have yet to adopt
+Pando React. For teams that have already adopted Pando React, this would be a
+non-breaking change since the tooling is internal and it's just adding some deps
+and a config file.
+
+If we approach this via the CLI promoted above, it should be even easier for teams
+to start using Pando without doing any work.
 
 # Alternatives
 
-What other designs have been considered? What is the impact of not doing this?
+## Pando team specific
+
+The alternative is to continue to do what we are doing now which is basically
+doing what Panda already does so we can streamline the process and make it easier
+by just migrating to Panda.
+
+## Company wide
+
+The alternative is to continue to do what we are doing now which is basically
+allowing all teams to do whatever they want for FE styling which has led to a
+lot of unnecessary complexity and issues on different levels across the company.
 
 # Adoption strategy
 
-If we implement this proposal, how will existing Pando developers adopt it? Is
-this a breaking change? Can we write a codemod? Should we coordinate with
-other projects or libraries?
+This would (and should) require us to officially sunset any internal UI libary outside of
+Pando (Classic, ps-mui, and anything else). This would be a good thing because it would
+allow us to focus on one library and make it the best it can be for the company.
 
-# How we teach this
+This process has already begun and is in the works for the next few months via our
+ops team.
```

Comment:
What process is this referring to?

Reply:
This is an internal strategy that is being followed through between the company, Pando, and the dev ops team to help create a SSoT for anything Design-system related for company standards (which is now Pando).

Reply:
Yes, basically, trying to solve the problem of "how do we make the FE more consistent across the Company like we do everything else?".

This update allows us to achieve that while ensuring teams can still style things how they like without us shipping 5 different libs within one page if that makes sense?